iroha/src/tx.rs

```diff
@@ -452,18 +421,46 @@ impl From<RejectedTransaction> for AcceptedTransaction {
     }
 }
 
+/// Query module provides `IrohaQuery` Transaction related implementations.
```

Comment:
Please link Query according to https://doc.rust-lang.org/rustdoc/linking-to-items-by-name.html

Reply:
Fixed